lib/capistrano/multiconfig.rb

```diff
@@ -27,6 +30,8 @@
     # Load defaults variables
     invoke 'load:defaults'
 
+    Rake.application["load:defaults"].extend(Capistrano::ImmutableTask)
```

Comment:
What is to reason to add this strict rule that disallow multiconfig users to modify `load:defaults` task?